extensions/sample-ui-plugin/src/client/styles.ts

```diff
@@ -69,3 +69,21 @@ export const output = css`
   font-size: 14px;
   font-weight: 400;
 `;
+
+export const title = css`
+  margin-top: 0;
+`;
+
+export const buttonBar = css`
+  display: flex;
+  flex-flow: row nowrap;
+  margin-top: 16px;
```

Comment:
Lets avoid using margin-top as much as possible. Usually, when an element is rearranged margin-bottom on the top element rather than margin-top on the element stacked below feels more natural.